syntax = "proto3";
package metrics;

message NodePerformance {
  map<string, double> mean_publish_period_ms = 1;
}
